corehq/apps/locations/models.py

```diff
@@ -834,6 +837,61 @@ def from_locations(cls, locations):
         related_locations = {loc_id for relation in relations for loc_id in relation}
         return related_locations - {l.location_id for l in locations}
 
+    @classmethod
+    def relation_distance_dictionary(cls, locations):
+        """Returns a dictionary of the related locations and their distance to each
+        location that's passed in if it has a relation.
+
+        If two locations that are passed in that have a relation. That relation is ignored.
```

Comment:
Why are those relations ignored?  I'm having a little trouble understanding the big picture of how this is used - is this generically useful or is it particular to making updates?

Reply:
My intention for this fixture was to provide the minimum distance from all assigned locations to the locations that aren't assigned. This method was intended as a helper to give the locations that are related to the locations passed in and it's distance. The fixture would then take whatever the minimum distance is and write that as an attribute to the fixture.

For example if you are assigned to "Boston" and "Cambridge" and those two locations have a LocationRelation that has a distance of 364 and "Cambridge" has some relation to "Somerville"  with a distance of 13 smoots, you would receive Boston and Cambridge in your normal restore and Somerville in your related locations with a distance of 13.

Reply:
gotcha, so it's distance from related locations to the nearest assigned location?  Makes sense to exclude distances between assigned locations then.